cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml

```diff
@@ -58,7 +58,7 @@ spec:
             timeoutSeconds: 5
           command:
             - /heapster
-            - --source=kubernetes.summary_api:''
+            - --source=kubernetes.summary_api:?kubeletPort=10250&kubeletHttps=true&insecure=true
```

Comment:
what kind of credential is going to be sent over this unverified connection?

Reply:
service account bearertoken without audience binding I imagine.

Reply:
Yes, it's bearer token
https://github.com/kubernetes/heapster/blob/b0a73e1c64a2787242368b55efdb542a632ef726/metrics/sources/kubelet/configs.go

Reply:
I think that would require changes in metric-server to use seperate token for scraping nodes, other then communicating with apiserver.

Reply:
Yes, it would. I think a code change might be required anyway, to provide a fallback when webhook authn/z isn't set up on the node.